test/card-test.js

```diff
@@ -92,15 +91,15 @@ describe('cards interactions', function () {
     assert.notEqual(card.title, undefined);
   });
 
-  // it('document contains close, expand, and save buttons', function () {
-  //   let card = new Card({id: 1, context: document.body, modal: false});
-  //   let msg1 = 'document contains close button';
-  //   let msg2 = 'document contains expand button';
-  //   let msg3 = 'document contains save button';
-  //   assert.notEqual(card.closeButton, undefined, msg1);
-  //   assert.notEqual(card.saveButton, undefined, msg2);
-  //   assert.notEqual(card.fullscreenButton, undefined, msg3);
-  // });
+  it('document contains close, expand, and save buttons', function () {
```

Comment:
Testing for the existence of elements within an instance of the `Card` class does not test behavior, but instead tests the definition of the `Card` class. This definition could change in the future and still remain a valid `Card`, however, this test would indicate a change in structure is invalid. I suggest removing this test.